website/snippets/quickstarts/change-way-model-materialized.md

```diff
@@ -54,10 +54,11 @@ By default, everything gets created as a view. You can override that at the dire
 
     </File>
 
-4. Enter the `dbt run` command. Your model, `customers` should now build as a view.
+4. Enter the `dbt run` command. Your model, customers should now build as a view. 
+   - BigQuery users should run `dbt run --full-refresh` (mentioned in step 5) to ensure the materialization changes are fully applied
```

Comment:
Are we saying they shouldn't `dbt run` and instead should `dbt run --full refresh`? If so, we might want to be more specific around that:
```suggestion
   - BigQuery users need to run `dbt run --full-refresh` instead of `dbt run` to full apply materialization changes.
```